### PR TITLE
QueryApp: enhance documentation of local installation via local::lib

### PR DESCRIPTION
**Pull request by mspncp**
@levitte this is a little "thank you" for you for fixing my addrev problems and pointing me to local::lib. Hope you like it.

### CODE REVIEW

**Comment by mspncp** [comment]
Best viewed [in a markdown viewer](https://github.com/openssl/tools/blob/00f7cc972d624be7494c3d2b351896a42405e38a/QueryApp/README.md).

File: QueryApp/README.md [hunk 1]
```diff
@@ -55,11 +55,24 @@ is slightly different:
...
+The local installation method works best if the `PERL5LIB` environment variable
+(among others) is set correctly in the shell profile. A comprehensive guide how
+to do it correctly can be found in the manual page (`perldoc local::lib`,
+or online at [local::lib](https://metacpan.org/pod/local::lib)).
 
-    eval "`perl -I$HOME/perl5/lib/perl5 -Mlocal::lib`"
+Essentially, it boils down to adding the following line to your shell profile
+(e.g. `.bash_profile`, `.bashrc` for bash):
+
+    eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"
```
Comment (levitte) [inline QueryApp/README.md]: Why is the bashism necessary? For a simple thing like this, back quote works uniformly in quite a number of shells...

**Comment by mspncp** [comment]
Is it a bashism? I didn’t know, I always thought backquotes were legacy syntax. I took the example from the manual page, but if you prefer the backquotes I‘ll revert it

**Comment by levitte** [comment]
So it appears I'm ill informed of what progress has happened...

https://unix.stackexchange.com/questions/126927/have-backticks-i-e-cmd-in-sh-shells-been-deprecated

File: QueryApp/README.md (hunk 1 above)
Comment (levitte) [reply QueryApp/README.md]: Never mind, `$()` turns out to have been adopted for POSIX, and I didn't know...

**Comment by mspncp** [comment]
Again, you were slightly faster: I was going to say: "If bashism means that it's not in the posix standard", then it's not a bashism:

http://pubs.opengroup.org/onlinepubs/9699919799/

**Comment by mspncp** [comment]
Argh! Frames should be deprecated, too! Here's the correct link:

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03

**Comment by mspncp** [comment]
ping

**Comment by levitte** [comment]
Wha...?  Oh, right, only OMC can merge.  Sorry, forgot...

**Comment by levitte** [comment]
Merged.

f629412298e2740a6a2ee0b8db183462667f5aee QueryApp: enhance documentation of local installation via local::lib

**Pull request closed without merging**